src/System.Text.Json/tests/Utf8JsonReaderTests.MultiSegment.cs

```diff
@@ -562,5 +565,53 @@ public static void CheckOnlyOneOfValueSpanOrSequenceIsSetSingleValue(string json
                 Assert.True(json.ValueSequence.IsEmpty);
             }
         }
+
+        [Theory]
+        [MemberData(nameof(SingleLineCommentData))]
+        public static void ConsumeSingleLineCommentMultiSpanTest(string expected)
+        {
+            string jsonData = "{" + expected + "}";
```

Comment:
> This is because of missing overload `Encoding.UTF8.GetBytes(string, int, int)` on that platform? Please advise.

Encode the whole data to UTF-8 bytes up front to avoid having to call the `string, int, int` overload of `Encoding.UTF8.GetBytes` within the loop.

```C#
byte[] dataUtf8 = Encoding.UTF8.GetBytes(jsonData);
ReadOnlySequence<byte> sequence = JsonTestHelper.GetSequence(dataUtf8, 1);

for (int i = 0; i < dataUtf8.Length; i++)
{
   var state = new JsonReaderState(options: new JsonReaderOptions { CommentHandling = JsonCommentHandling.Allow });

   var json = new Utf8JsonReader(sequence.Slice(0, i), isFinalBlock: false, state);
   VerifyReadLoop(ref json, expected);

   state = json.CurrentState;
   json = new Utf8JsonReader(sequence.Slice(state.BytesConsumed), isFinalBlock: true, state);
   VerifyReadLoop(ref json, expected);
}

```